cristin-import/src/main/java/no/unit/nva/cristin/CristinMapper.java

```diff
@@ -0,0 +1,81 @@
+package no.unit.nva.cristin;
+
+import java.net.URI;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Stream;
+import no.unit.nva.model.AdditionalIdentifier;
+import no.unit.nva.model.EntityDescription;
+import no.unit.nva.model.Publication;
+import no.unit.nva.model.PublicationDate;
+
+public class CristinMapper {
```

Comment:
A template of a mapping the two versions of publication. The basic pattern will be 
```
streamOfCristinObjects.map(input->new CristinMapper(input)).map(mapper->mapper.generatePublication())
```